tasks/generate.py

```diff
@@ -146,3 +146,17 @@ def publish_release(ctx, version, user, pypi_name):
     print(' ', ','.join(emails))
     print()
     print('And announce it on twitter adding the #pytest hash tag.')
+
+
+@invoke.task(help={
+    'version': 'version being released',
+})
+def changelog_draft(ctx, version):
+    check_call(['towncrier', '--draft', '--version', version])
+
+
+@invoke.task(help={
+    'version': 'version being released',
+})
+def changelog_update(ctx, version):
```

Comment:
makes sense as well, in that case we should not have a draft option, but a write option in order to opt in instead of out

i think i should invoke the write version from the prerelease step